### ISSUE TITLE
missing readme file in HW8 

### ISSUE THREAD

**Issue by fedhere** [AbdulRehmanUDEL/DSPS_ARehman#5]
are you putting in your readme files?? if now you are loosing a ton of points for no reason!

**Comment by fedhere** [comment]
YOur readme should be called just 

README.md - this way they are automatically displayed. PLease change them.

**Comment by AbdulRehmanUDEL** [comment]
Yes, I am putting readme files. Sorry, I did not know that they should be called with specific names. I have changed them.

**Comment by fedhere** [comment]
ok
___________________________________________________
federica b. bianco
*http://fbb.space/ <http://fbb.space/>*
____________________________________________________


                                     -                         _`
                                   `+*-                       ;*;
                                  `o**=                  `   :**o.
                     `.~.`        ;*ooo`                ~~  .o*oo=
                    `-----~       =oio*                ~--` :ooo**.^`
 `
      ;`            .------`      :oee;               `---^ :e*i!?oi^
+_  -io**=
      $$i.        `.:;_---`   ~`  `+=_                `---` .*ee?!i+
i: -eo*oe;       .~^
      $OOOe_      -__++_-`   ^-.   ^ ``                .~_:_.-*!!e*~
~_~i?eo*o~      `---^
      $OOOOO?;`   -----^     `-^  ^__:-`       ^~      ~;.    .*ee:
-. -!?eo**       ^----
      $OOOOOOO$i. `~-^        ^.^`^--_^        _-    ^=- `~:;;+++^  --
  _!?***:        --~`
      :_;o?OOOOO$*-      `_:-~^    ``     ~.  ~:-   ;o.:+:~`~i+++_;_
  .ie**+
           ^;!OOOOO!_   :i^               i:  --` ^*e+:.=       ~o`
^::``.`
              .=%OOOO?_o:                 ;. `..~i$o_^..^+`
`;++++___.
                 _eOOOOo^               .:ie%?!e?*_;+_oo=^__.        ^
-i*o_  `
                   ~o$OO$*:..-^  .-::=e$$!=-`    `io**eo*:  ^`
~;:~*oii~ ~::~
                     ^?OOOOOOOO%$OOOO%o:^    +_  :*o*e*o*+    `.:-
:=+~.::. `++++.
                      `;i$O?%O%=io*e?$%e=~   oo^ +*oo*oo*;
.io*i^^_--`    ^++++_     ^-~
                         _; `=Oi`     `~;o!=^~=^ _**ooo**_    ~*ioo=
^.^      ;+++_     ---`
                              ;%%-         ~o=`  `oeoo:.;~     i*oii^
       `:++-     .--
                                +$~          ..   ;*e;          _iii:
         `.~      .^
                          ~--.  ^;%=~_;++++:`     ^*_
                         ~+:--``-`^;o!*+-  `:;`    `
             `.
                        `;++_-. ^    `.-_+:~  _;~
             --^
                        ^:++_-^     ^-;_    ..``~-.
             -_-.
                        `_;;-~      .-_-       ^---``
              __--
                         --_~       `--`       ^--;_-~
              ----^
                         .-^          .         ~_+_--^
              ----
                          `                      -;---.
              ^--^
                                                 `_---~
               .`
                                                  ^---.
          ``
                                                   `~-`
         `++.
                                                     .
         ^++_   -~

          ~:-   ~~

            `

             ^.

             ^_.

              --~

              ~-^

              `^

Please consider the environment before printing this e-mail
